### PR TITLE
Rename PAClab to ARG-V

### PR DESCRIPTION
**Pull request by psybers**
This also includes any filenames and paths with `paclab` in it.

### CODE REVIEW

File: argv_backend/transformrunner.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
 """
 class TransformRunner(TR):
     def run(self):
-        self.transformer_path = config('PACLAB_TRANSFORM_PATH', default='transformer')
+        self.transformer_path = config('TRANSFORMER_PATH', default='transformer')
         self.config_path = os.path.join(tempfile.gettempdir(), 'paclab-' + str(self.transformed_project.pk))
```
Comment (swflint) [inline argv_backend/transformrunner.py]: Do you want to still prefix the config path with `paclab` here?

File: argv_backend/fixtures/argv_install.json
Comment (swflint) [inline argv_backend/fixtures/argv_install.json:1]: The name `paclab` will need changed to `argv` for install of the backend to run.

File: argv/settings.py
Comment (swflint) [inline argv/settings.py:1]: Check that `ROOT_URLCONFIG` and `WSGI_APPLICATION` are correct here.

**Review by swflint — CHANGES_REQUESTED** [review]
You might want to do a `grep -R` for `paclab`.  I think I caught most of it in my comments, but it seems to work now.

File: argv_backend/transformrunner.py (hunk 1 above)
Comment (psybers) [reply argv_backend/transformrunner.py]: No.

**Review by swflint — APPROVED** [review]
It seems to work with these changes.

**Pull request merged**